src/client/components/Inputs/Select/Select.tsx

```diff
@@ -29,21 +34,22 @@ const Select: React.FC<SelectProps> = (props) => {
         multiValue: ({ isDisabled }) => classNames('select__multiValue', { isDisabled }),
         multiValueLabel: ({ isDisabled }) => classNames('select__multiValueLabel', { isDisabled }),
         multiValueRemove: ({ isDisabled }) => classNames('select__multiValueRemove', { isDisabled }),
-        option: ({ isFocused, isSelected }) => classNames('select__option', { isFocused, isSelected }),
+        option: ({ isFocused, isSelected }) => classNames('select__option', { isFocused, isSelected, toggleAll }),
         placeholder: () => `select__placeholder`,
         singleValue: () => 'select__singleValue',
         valueContainer: () => 'select__valueContainer',
       }}
       closeMenuOnSelect={!isMulti}
-      components={{ ClearIndicator, DropdownIndicator, IndicatorsContainer, IndicatorSeparator: null }}
+      components={components}
+      hideSelectedOptions={hideSelectedOptions}
```

Comment:
If it is always set to false, when we pass the 'isMulti' prop but not the 'toggleAll' prop, it would behave like this:

https://github.com/openforis/fra-platform/assets/41337901/b7baedda-e584-41bf-a826-88eeee2fbd7d

Instead of how it currently is:

https://github.com/openforis/fra-platform/assets/41337901/a0577ca9-3731-4dac-a416-cb36ec954e3c

Is that what we want?

Reply:
I've been looking into setting the color of the checkbox in the indeterminate state but it is not possible to do that by simply adding an scss attribute. I would need to unset default styles and create custom styles from scratch. Should we do that later on when we add the simple checkbox option?

Reply:
It would be what I mentioned in the other comment about maybe adding a 'variant' prop to the checkbox. And when you pass for instance variant='simple', you would get this simple checkbox without any extra borders or other elements.